common/test/FormattingUtils.js

```diff
@@ -0,0 +1,42 @@
+const test = require("tape");
+const utils = require("../FormattingUtils");
+const BigNumber = require("bignumber.js");
+
+// Please recommend a better place/way to do this. Technically this
+// does not depend on web3 or truffle, but can include that for consistency.
+test("FormattingUtils", t => {
```

Comment:
Should we add this to CI? It makes sense perhaps to have a `test/` folder for each `package` and then we can add `test` folders to CI individually. @mrice32 probably knows better how to do that, but I do think it makes sense to have separate `test` folders per package

Reply:
Each package has its own `test` npm script. That's what gets run by ci. If you want to add tests to this package (which is a great idea!), we should do so by adding to that `test` npm script. Here's [an example](https://github.com/UMAprotocol/protocol/blob/277d2419b8cc7f93877f417cd0fd32007cf93b31/packages/monitors/package.json#L31).

We do use truffle everywhere else, but truffle basically uses mocha under the hood (that's where all the syntax in our existing tests come from). We can open an issue to discuss _not_ using mocha for non truffle tests (there may be good reasons for this!), but in the absence of a reason to use a different framework, I think we should default to mocha just for the sake of consistency.